contexts/compound-and-earn-context.js

```diff
@@ -82,8 +96,46 @@ export function CompoundAndEarnProvider({ children }) {
     }
   }
 
+  const getUserPools = async () => {
+    if (account && pools && gauges) {
+      const formatedPools = await Promise.all(pools.map(async (pool) => {
+        let userLP = 0;
+        if (pool.kind === 'Snowglobe') {
+          const gauge = gauges.find((item) => item.gaugeAddress.toLowerCase() ===
+            pool.gaugeInfo.address.toLowerCase());
+          if (gauge) {
+            const snowglobeContract = new ethers.Contract(pool.address,
+              SNOWGLOBE_ABI, library.getSigner());
+
+            const totalSupply = await snowglobeContract.totalSupply() / 1e18;
+            const snowglobeRatio = (await snowglobeContract.getRatio()) / 1e18;
```

Comment:
what is `snowglobeRatio` for?

Reply:
> what is `snowglobeRatio` for?

I use that to calculate the LP holdings of the user using only the sPGL that he has either in the snowglobe contract and gauge contract.
It means how many one PGL is to 1 sPGL. (normally this number is 1.2 , 1.4 something and grows overtime).